src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -164,11 +167,12 @@ export class ExposureNotificationService {
   async startKeysSubmission(oneTimeCode: string): Promise<void> {
     const keys = await this.backendInterface.claimOneTimeCode(oneTimeCode);
     const serialized = JSON.stringify(keys);
-    console.log(serialized);
     try {
-      await this.secureStorage.set(SUBMISSION_AUTH_KEYS, serialized, {});
+      await this.secureStorage.set(SUBMISSION_AUTH_KEYS, serialized, {
+        accessible: ACCESSIBLE.WHEN_UNLOCKED_THIS_DEVICE_ONLY,
+      });
     } catch (error) {
```

Comment:
No issues with your catching of the exception, but it did draw my attention to the overall flow of this function.

Can you confirm my interpretation of this?
```

1: Trade OTC for AuthKeys
2: Store AuthKeys into secureStorage
2.5 if 2 throws exception (log, and continue)
3: Put phone (exposureStatus) into 'diagnosed' state

```
If 2.5 is a passthrough (apologies, JS isn't my hometurf), then this would create a pretty faulty condition where the phone was in diagnosed state, and would have no ability to uploadKeys. If the catch ejects from function, no worries, and ignore the thought.

Nothing that needs addressing now, as I'm going to assume this is a very rare situation (if in fact it works the way I thinkg), but figure it's worth flagging.

Reply:
We don't have any analytics at all on prod. Doing try / catch here will cause the app cannot upload keys in the next cycle. I would prefer to remove try / catch. Set to storage needs to work 100% of the time, I think.

Reply:
@henrytao-me go ahead with removing the try catch

Reply:
The `try/catch` in this case was not added during this PR. Just wondering if it's necessary to fix it at this point, or should it be a new issue?